chroma_core/management/commands/dev_setup.py

```diff
@@ -17,18 +17,11 @@
 
 class Command(BaseCommand):
     option_list = BaseCommand.option_list + (
-        make_option(
-            "--no-bundles",
-            action="store_true",
-            dest="no_bundles",
-            default=False,
-            help="Do not load any bundles, instead create dummies",
-        ),
+        make_option("--no-bundles", action="store_true", dest="no_bundles", default=False, help="Vestigital Setting"),
```

Comment:
What happens if we remove this?

Reply:
probably nothing, since I think I've gotten everywhere that called it.